song_story/core/nlu/spacy.py

```diff
@@ -0,0 +1,66 @@
+import spacy
+import re
+import string
+import json
+
+
+class SpacyModel:
+    """
+    This is class for spacy language model
+    and its associated functions
+    """
+
+    def __init__(self, size="md"):
+
+        # Model size
+        if size in ["md", "sm", "lg"]:
+            model_size = "en_core_web_{}".format(size)
+        # Load NLP model
+        self.nlp = spacy.load(model_size, disable=["tagger", "ner"])
+        # Load Clist
```

Comment:
remove commented code if they are of no use

Reply:
Removed the code